src/transformers/trainer.py

```diff
@@ -2177,11 +2177,18 @@ def _load_best_model(self):
         logger.info(f"Loading best model from {self.state.best_model_checkpoint} (score: {self.state.best_metric}).")
         best_model_path = os.path.join(self.state.best_model_checkpoint, WEIGHTS_NAME)
         best_safe_model_path = os.path.join(self.state.best_model_checkpoint, SAFE_WEIGHTS_NAME)
+        adapter_model_path = os.path.join(self.state.best_model_checkpoint, "adapter_model.bin")
```

Comment:
it can also be safetensor ckpts, right?

Reply:
Maybe adding `best_safe_adapter_model_path` should serve the purpose?

Reply:
perfect, will refactor that a bit